src/state-prototypes.js

```diff
@@ -73,6 +74,10 @@ const stateSchema = {
       type: 'array',
       items: cellSchema,
     },
+    pinnedCells: {
+      type: 'array',
+      items: cellSchema,
```

Comment:
I think this would create redundancy. We would be storing information about pinned cells at 2 places (`cells` and `pinnedCells`). The idea was to either store a boolean in each cell or make a pinnedCell key value pair where we just store the cell ids of those cells which are pinned.

Reply:
Yes, I was also thinking this, but I added both. 
I think @hamilton can guide us on this.